css/structure/jquery.mobile.core.css

```diff
@@ -279,6 +279,7 @@ div.ui-mobile-viewport {
 	padding: 0;
 }
 .ui-btn-inline {
+	max-width:100% ;
```

Comment:
@arschmitz 

Agreed. Let's fix this in the selectmenu CSS.

I think we should also look at old (closed) tickets because I know we have been dealing with select width issues in the past and don't want regressions. Might be good to add a unit test for this.

We should do some testing with ```max-width: 100%;``` for the selectmenu button in the ui-1-12 branch after my CSS update (no PR yet) landed in there.